srcalc/src/main/java/gov/va/med/srcalc/service/DefaultAdminService.java

```diff
@@ -31,6 +31,14 @@
     private final RuleDao fRuleDao;
     private final ProcedureDao fProcedureDao;
     
+    /**
+     * Constructs an instance.
```

Comment:
How about: "Constructs an instance that will use the given DAOs to query and update the database."